src/ensembl/src/styles/_settings.scss

```diff
@@ -17,43 +18,40 @@ $global-button-cursor: auto;
 
 // Colour Palette
 // ---------
+$shadow-color: rgba(0, 0, 0, 0.4);
 
-$ens-black: #17354e;
-$ens-shadow-color: rgba(0, 0, 0, 0.4);
+$light-blue: #e5f5ff; // text highlight
+$blue: #33adff; // blue on black
+$dark-blue: #0099ff; // blue on white
 
-$ens-light-blue: #e5f5ff; // text highlight
-$ens-blue: #33adff; // blue on black
-$ens-dark-blue: #0099ff; // blue on white
+$light-grey: #f1f2f4;
+$medium-light-grey: #d4d9de;
+$grey: #b7c0c8;
+$medium-dark-grey: #9aa7b1;
+$dark-grey: #6f8190;
 
-$ens-light-grey: #f1f2f4;
-$ens-medium-light-grey: #d4d9de;
-$ens-grey: #b7c0c8;
-$ens-medium-dark-grey: #9aa7b1;
-$ens-dark-grey: #6f8190;
+$red: #d90000;
 
-$ens-red: #d90000;
+$green: #47d147;
 
-$ens-green: #47d147;
 
-$ens-white: #fff;
-
-$global-box-shadow: $ens-medium-light-grey;
+$global-box-shadow: $medium-light-grey;
 
 /* stylelint-disable */
 :export {
-  ens-black: $ens-black;
-  ens-blue: $ens-blue;
-  ens-light-blue: $ens-light-blue;
-  ens-dark-blue: $ens-dark-blue;
-  ens-red: $ens-red;
-  ens-dark-grey: $ens-dark-grey;
-  ens-medium-dark-grey: $ens-medium-dark-grey;
-  ens-grey: $ens-grey;
-  ens-medium-light-grey: $ens-medium-light-grey;
-  ens-light-grey: $ens-light-grey;
-  ens-green: $ens-green;
-  ens-white: $ens-white;
-  ens-shadow-color: $ens-shadow-color;
+  black: $black;
+  blue: $blue;
+  light-blue: $light-blue;
+  dark-blue: $dark-blue;
+  red: $red;
+  dark-grey: $dark-grey;
+  medium-dark-grey: $medium-dark-grey;
+  grey: $grey;
+  medium-light-grey: $medium-light-grey;
+  light-grey: $light-grey;
+  green: $green;
+  white: $white;
+  shadow-color: $shadow-color;
```

Comment:
Should off-white also go to exported values and to storybook then?

Reply:
As discussed offline, let's merge this first and then see if we need to add extra colours to storybook (and whether we do indeed need two white colours)

Reply:
OK. Could you take a look at the conflict? It looks to me like a CSS classname has been changed, so the new classname is using the old colour name whereas the old classname has the new colour name! (which is what I was afraid might happen)

Reply:
Yes, that's exactly it. The class `currentDrawerView` was renamed to `trackHighlighted`, and at the same time the background colour variable name changed from `$ens-light-blue` to `$light-blue`.

I pushed a commit resolving this conflict.